[IMP] Inventory: Reordering rules 001 update (#9788)
odoo/documentation: content/applications/inventory_and_mrp/inventory/product_management/product_replenishment/reordering_rules.rst

   not reserved for any specific |SO|, and can be used as needed.

   Another key difference is that replenishment orders created by the Replenish on Order route are
   linked to the original |SO| by a smart button at the top of the order. When using the 0/0/1
   reordering rule, a replenishment order is created, but is not linked to the original |SO|.

   See the :doc:`Replenish on Order (MTO) <mto>` documentation for a full overview of the MTO route.

To create a 0/0/1 reordering rule, navigate to :menuselection:`Inventory app --> Products -->
Products`, and select a product.

At the top of the product's page, click the :icon:`fa-refresh` (Reordering Rules) smart button to
open the reordering rules page for the product. On the page, click :guilabel:`New` to begin
configuring a new reordering rule.

In the :guilabel:`Route` field of the new reordering rule, select the route the rule should use to
replenish the item. For example, if the product should be purchased from a vendor, select the
:guilabel:`Buy` route.

In the :guilabel:`Min Quantity` field and :guilabel:`Max Quantity` field, leave the values set to

> `0.00`. In the :guilabel:`To Order` field, enter a value of `1.00`.

Reviewer: To my knowledge, we don't manually enter `1 ` in the To Order field. We'd create a sales order so that forecasted quantity is -1, and Odoo automatically increments the To Order value to 1.
```suggestion
`0.00`. With this set up, once an order is placed, the reordering rule is triggered to bring the negative stock value back to 0.
```
Also, feel free to reword this as you see fit! :D

Author: I just created a new reordering rule to confirm, and the default To Order value is 0.

.. image:: reordering_rules/001-rule.png
   :align: center
   :alt: A 0/0/1 reordering rule.

With the reordering rule configured using these values, each time an |SO| causes the forecasted
quantity of the product to fall below the :guilabel:`Min Quantity` of `0.00`, the selected
:guilabel:`Route` is used to replenish the product in one-unit increments, back up to the
:guilabel:`Max Quantity` of `0.00`.

.. example::
   A picture frame is configured with a 0/0/1 reordering rule that uses the *Buy* route. Zero units
   of the picture frame are kept on-hand at any given time.

>    A purchase order (PO) is confirmed for one unit of the picture frame, which causes the forecasted

Reviewer: ```suggestion
   A sales order is confirmed for one unit of the picture frame, which causes the forecasted
```

- :ref:`Visibility days <inventory/product_management/visibility-days>`
- :ref:`Route <inventory/product_management/route>`

0/0/1 reordering rule
---------------------

The *0/0/1* reordering rule is a specialty rule used to replenish a product that is not kept
on-hand, each time a sales order (SO) is confirmed for that product.

.. important::
   The 0/0/1 reordering rule is similar to the *Replenish on Order (MTO)* route, in that both
   workflows are used to replenish a product upon confirmation of an |SO|.

>    The main difference between the two methods is that the Replenish on Order route automatically

Reviewer 2: Italicize the route's name since you are referencing it exactly :)
```suggestion
   The main difference between the two methods is that the *Replenish on Order* route automatically
```

>    reserves the product for the |SO| that caused it to be replenished. This means the product cannot

Reviewer 3: ```suggestion
   reserves the product for the |SO| that caused it to be replenished. This means the product **cannot**
```
SUGGESTION: just for added emphasis

   **cannot** be used for a different |SO|.

   The 0/0/1 reordering rule does not have this limitation. A product replenished using the rule is
   not reserved for any specific |SO|, and can be used as needed.

>    Another key difference is that replenishment orders created by the Replenish on Order route are

Reviewer 2: Here too please
```suggestion
   Another key difference is that replenishment orders created by the *Replenish on Order* route are
```

   linked to the original |SO| by a smart button at the top of the order. When using the 0/0/1
   reordering rule, a replenishment order is created, but is not linked to the original |SO|.

   See the :doc:`Replenish on Order (MTO) <mto>` documentation for a full overview of the MTO route.

To create a 0/0/1 reordering rule, navigate to :menuselection:`Inventory app --> Products -->
Products`, and select a product.

At the top of the product's page, click the :icon:`fa-refresh` :guilabel:`Reordering Rules` smart

> button to open the :guilabel:`Reordering Rules` page for the product. On the page, click

Reviewer 3: ```suggestion
button to open the :guilabel:`Reordering Rules` page for the product. On the resulting page, click
```

In the :guilabel:`Min Quantity` field and :guilabel:`Max Quantity` field, leave the values set to
`0.00`. In the :guilabel:`To Order` field, enter a value of `1.00`.

.. image:: reordering_rules/001-rule.png
   :align: center
   :alt: A 0/0/1 reordering rule.

With the reordering rule configured using these values, each time an |SO| causes the forecasted
quantity of the product to fall below the :guilabel:`Min Quantity` of `0.00`, the selected
:guilabel:`Route` is used to replenish the product in one-unit increments, back up to the
:guilabel:`Max Quantity` of `0.00`.

.. example::
   A picture frame is configured with a 0/0/1 reordering rule that uses the *Buy* route. Zero units
   of the picture frame are kept on-hand at any given time.

   An |SO| is confirmed for one unit of the picture frame, which causes the forecasted quantity to
   drop to `-1.00`. This triggers the reordering rule, which automatically creates a |PO| for one
   unit of the picture frame.

   Once the product is received from the vendor, the forecasted quantity of the picture frame returns
   to `0.00`. There is now one picture frame on-hand, but it is not reserved for the SO which

>    triggered its purchase. It can be used to fulfill that SO, or reserved for a different order.

Reviewer 3: just a suggestion, do you want to consider using the abbreviate |SO| formatting for these last two mentions, or just leave them as they are. Totally fine either way -- completely up to you -- just thought it was worth mentioning for your consideration.

>    Once the product is received from the vendor, the forecasted quantity of the picture frame returns
>    to `0.00`. There is now one picture frame on-hand, but it is not reserved for the |SO| which
>    triggered its purchase. It can be used to fulfill that |SO|, or reserved for a different order.

Reviewer 2: Line too long (over 100 chars) :)
```suggestion
   Once the product is received from the vendor, the forecasted quantity of the picture frame
   returns to `0.00`. There is now one picture frame on-hand, but it is not reserved for the |SO|
   which triggered its purchase. It can be used to fulfill that |SO|, or reserved for a different
   order.
```
